tests/python/gaia-ui-tests/gaiatest/apps/search/app.py

```diff
@@ -33,15 +40,9 @@ def go_to_url(self, url):
 
     @property
     def history_items_count(self):
+        Wait(self.marionette).until(expected.element_present(*self._history_locator))
```

Comment:
Yes, I had a case where the browser wasn't there at all and then a check with history_items_count == 0 would still pass, even though the browser wasn't there.